src/Modules/ModParser.lua

```diff
@@ -3643,6 +3643,9 @@ local specialModList = {
 	["every 8 seconds, gain avatar of fire for 4 seconds"] = {
 		flag("Condition:HaveVulconus"),
 	},
+	["trigger socketed curse spell when you cast a curse spell, with a ([%d%.]+) second cooldown"] = {
+		flag("Condition:HaveVixensEntrapment"),
+	},
```

Comment:
Vixen's Entrapment needs updating in PoB
![image](https://user-images.githubusercontent.com/31533893/210465339-f27cf27c-70de-469b-88b8-f8d5aad3011f.png)

Reply:
I've been trying to get the exporter to work for a while now, unsuccessfully. Can I just change this specific line in gloves.lua?

Reply:
Yes unique are all manually done.